### PR TITLE
Crowd 2.1.3

### PR DESCRIPTION
**Pull request by deepy**
This contains a fix for the JAXB issue.

### CODE REVIEW

File: crowd.properties [hunk 1]
```diff
@@ -2,13 +2,20 @@ category=Integration
...
 devVersion=
-publicVersions=2.1.2
+publicVersions=2.1.2,2.1.3
 archivedVersions=1.0,2.0,2.1,2.1.1
 privateVersions=
 
 defaults.mavenGroupId=cm.xd.sonar-plugins
 defaults.mavenArtifactId=sonar-crowd-plugin
 
+2.1.3.description=Support SonarQube 6.7-7.7
```
Comment (colin-mueller-sonarsource) [inline crowd.properties]: If this version only supports 7.9+, you should probably update the description!

File: crowd.properties [hunk 2]
```diff
@@ -2,13 +2,20 @@ category=Integration
...
+2.1.3.sqVersions=[7.9,LATEST]
+2.1.3.date=2019-06-03
```
Comment (colin-mueller-sonarsource) [inline crowd.properties]: 🤔 This date was a month ago!

Comment (deepy) [reply crowd.properties]: I might've been thinking a bit too far ahead :flushed:

**Comment by deepy** [comment]
Thank you!
I've updated the PR!

**Pull request merged**